lib/ble.toit

```diff
@@ -278,13 +278,18 @@ class RemoteCharacteristic extends RemoteReadWriteElement_ implements Attribute:
 
   /**
   Writes the value of the characteristic on the remote device.
+
+  If $flush is true, waits until the data has been written. This is the default for
```

Comment:
Flush seems to just default to false all the way down to the primitive. It seems quite convoluted. Is it really necessary?